### PR TITLE
feat: option scalps

### PR DESCRIPTION
**Pull request by Aercwarden**

### CODE REVIEW

File: packages/ui/src/MenuItems.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import React from "react";
 import { Menu } from "@headlessui/react";
```
Comment (witherblock) [inline packages/ui/src/MenuItems.tsx:1]: Why change formatting for file?

File: packages/ui/src/Tab.tsx [hunk 2]
```diff
@@ -18,15 +18,9 @@ const Tab: FC<TabProps> = (props: TabProps) => {
```
Comment (witherblock) [inline packages/ui/src/Tab.tsx:18]: Why change formatting for file?

File: packages/ui/src/icons/CheckedIcon.tsx [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import React, { SVGProps } from "react";
+import React, {  SVGProps } from "react";
```
Comment (witherblock) [inline packages/ui/src/icons/CheckedIcon.tsx:1]: Why change formatting?

File: packages/ui/src/icons/ContentCopyIcon.tsx [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import React, { SVGProps } from "react";
+import React, {  SVGProps } from "react";
```
Comment (witherblock) [inline packages/ui/src/icons/ContentCopyIcon.tsx:1]: Why change formatting?

File: packages/ui/src/icons/DropdownArrowIcon.tsx [hunk 5]
```diff
@@ -1,4 +1,4 @@
-import React, { SVGProps } from "react";
+import React, {  SVGProps } from "react";
```
Comment (witherblock) [inline packages/ui/src/icons/DropdownArrowIcon.tsx:1]: Why change formatting?

File: packages/ui/src/icons/InsuredPerpsIcon.tsx [hunk 6]
```diff
@@ -1,4 +1,4 @@
-import React, { SVGProps } from "react";
+import React, {  SVGProps } from "react";
```
Comment (witherblock) [inline packages/ui/src/icons/InsuredPerpsIcon.tsx:1]: Why change formatting?

File: packages/ui/src/icons/LongStraddleIcon.tsx [hunk 7]
```diff
@@ -1,17 +1,8 @@
...
-    xmlns="http://www.w3.org/2000/svg"
-    {...props}
-  >
-    <path
-      d="M3.077 9.23V3.078L6.923 9.23h6.538l3.462-5.723V9.23H3.077Z"
-      fill="#C3F8FF"
-    />
+  <svg width={20} height={20} fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
+    <path d="M3.077 9.23V3.078L6.923 9.23h6.538l3.462-5.723V9.23H3.077Z" fill="#C3F8FF" />
     <path d="M10 11.539 8.846 8.462h2.308L10 11.539Z" fill="#FF617D" />
```
Comment (witherblock) [inline packages/ui/src/icons/LongStraddleIcon.tsx:6]: Why change formatting?

File: packages/ui/src/icons/PegHedgeIcon.tsx [hunk 8]
```diff
@@ -1,23 +1,9 @@
...
-      strokeWidth={3}
-      strokeLinecap="round"
-    />
-    <path
-      d="M10 5.385A4.614 4.614 0 0 0 5.385 10 4.614 4.614 0 0 0 10 14.615 4.614 4.614 0 0 0 14.615 10 4.614 4.614 0 0 0 10 5.385Z"
-      fill="#C3F8FF"
-    />
+  <svg width={20} height={20} fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
+    <path d="M3.846 10h12.308" stroke="#8E8E8E" strokeWidth={3} strokeLinecap="round" />
+    <path d="M10 5.385A4.614 4.614 0 0 0 5.385 10 4.614 4.614 0 0 0 10 14.615 4.614 4.614 0 0 0 14.615 10 4.614 4.614 0 0 0 10 5.385Z" fill="#C3F8FF" />
```
Comment (witherblock) [inline packages/ui/src/icons/PegHedgeIcon.tsx:6]: Why change formatting?

File: packages/ui/stories/Menu.stories.tsx [hunk 9]
```diff
@@ -1,12 +1,11 @@
...
-import PegHedgeIcon from "../src/icons/PegHedgeIcon";
+import CheckedIcon from "../src/icons/CheckedIcon";
+import ContentCopyIcon from "../src/icons/ContentCopyIcon";
 import InsuredPerpsIcon from "../src/icons/InsuredPerpsIcon";
 import LongStraddleIcon from "../src/icons/LongStraddleIcon";
-import ContentCopyIcon from "../src/icons/ContentCopyIcon";
-import CheckedIcon from "../src/icons/CheckedIcon";
+import PegHedgeIcon from "../src/icons/PegHedgeIcon";
+import {  ComponentMeta } from "@storybook/react";
+import React from "react";
```
Comment (witherblock) [inline packages/ui/stories/Menu.stories.tsx:8]: Why change formatting?

File: packages/ui/stories/MenuItems.stories.tsx [hunk 10]
```diff
@@ -1,11 +1,10 @@
...
-
-import MenuItems, { ItemType } from "../src/MenuItems";
-import PegHedgeIcon from "../src/icons/PegHedgeIcon";
+import MenuItems, {  ItemType } from "../src/MenuItems";
 import InsuredPerpsIcon from "../src/icons/InsuredPerpsIcon";
 import LongStraddleIcon from "../src/icons/LongStraddleIcon";
+import PegHedgeIcon from "../src/icons/PegHedgeIcon";
+import {  Menu as HeadlessMenu } from "@headlessui/react";
+import {  ComponentMeta } from "@storybook/react";
+import React from "react";
```
Comment (witherblock) [inline packages/ui/stories/MenuItems.stories.tsx:7]: Why change formatting?

File: packages/ui/stories/Tab.stories.tsx [hunk 11]
```diff
@@ -1,7 +1,7 @@
 import AlarmIcon from "../../../apps/dapp/src/svgs/icons/AlarmIcon";
 import Tabs from "../src/Tab";
 import TabHeader from "../src/TabHeader";
-import { ComponentMeta } from "@storybook/react";
+import {  ComponentMeta } from "@storybook/react";
```
Comment (witherblock) [inline packages/ui/stories/Tab.stories.tsx:4]: Why change formatting?

**Review by witherblock — CHANGES_REQUESTED** [review]

File: apps/dapp/public/images/tokens/arb.svg [hunk 12]
```diff
@@ -0,0 +1,2 @@
...
...l-.046-28.727-96.103-149.684-35.654 60.5 92.774 150.043 33.544-19.438a14.87 14.87 0 0 0 5.49-10.825l-.005-1.869z"/><path fill="#FFF" d="m20.219 382.469 47.369 27.296 157.634-252.801-26.753-.706c-22.795-.322-47.35 5.604-58.62 24.001L50.334 319.004l-30.115 46.271v17.194zM316.202 156.964l-70.51.256L86.17 420.462l55.758 32.103 15.163-25.716z"/><path fill="#96BEDC" d="M452.65 156.601c-.59-14.746-8.574-28.245-21.08-36.104L256.28 19.692c-12.371-6.229-27.825-6.237-40.218-.004-1.465.739-170.465 98.752-170.465 98.752a44.953 44.953 0 0 0-6.711 3.975c-11.164 8.001-17.969 20.435-18.668 34.095v208.765l30.115-46.271-.263-161.083c.035-.589.109-1.169.214-1.741a14.834 14.834 0 0 1 5.983-9.456c.795-.569 172.682-100.064 173.228-100.337 4.04-2.029 9.175-2.053 13.22-.063l173.022 99.523a14.893 14.893 0 0 1 6.887 11.859v199.542c-.209 4.231-1.882 8.152-5.172 10.825l-33.544 19.438-17.308 10.031-61.864 35.852-62.737 36.357c-3.379 1.221-7.33 1.152-10.681-.21l-74.228-42.693-15.163 25.717 66.706 38.406c2.206 1.255 4.171 2.367 5.784 3.272 2.497 1.4 4.199 2.337 4.8 2.629 4.741 2.303 11.563 3.643 17.71 3.643a44.596 44.596 0 0 0 16.332-3.072l182.225-105.531c10.459-8.104 16.612-20.325 17.166-33.564V156.601z"/></svg>
```
Comment (witherblock) [inline apps/dapp/public/images/tokens/arb.svg:2]: Unoptimized

File: apps/dapp/src/components/common/DexScreenerChart/index.tsx [hunk 13]
```diff
@@ -0,0 +1,17 @@
...
+import { SingleChartProps } from './SingleChart';
+
+type ChartType = {
+  poolAddress: string;
+};
+
+// @ts-ignore
+const Chart = dynamic<ChartType>(() => import('./SingleChart.tsx'), {
+  ssr: false,
+});
```
Comment (witherblock) [inline apps/dapp/src/components/common/DexScreenerChart/index.tsx:11]: Add TODO fix

File: apps/dapp/src/components/common/TradingViewChart/index.tsx [hunk 14]
```diff
@@ -0,0 +1,17 @@
...
+import { SingleChartProps } from './SingleChart';
+
+type ChartType = {
+  symbol: string;
+};
+
+// @ts-ignore
+const Chart = dynamic<ChartType>(() => import('./SingleChart.tsx'), {
+  ssr: false,
+});
```
Comment (witherblock) [inline apps/dapp/src/components/common/TradingViewChart/index.tsx:11]: Add TODO fix

**Review by witherblock — CHANGES_REQUESTED** [review]

**Review by hellodespereaux — APPROVED** [review]

**Pull request merged**